packages/core/index.js

```diff
@@ -4,7 +4,6 @@ var pkg = require("./package.json");
 module.exports = {
   build: require("./lib/build"),
   create: require("./lib/commands/create/helpers"),
-  console: require("./lib/repl"),
```

Comment:
@gnidan I removed this because it wasn't being used anywhere, but now I see it is in the truffle scenario tests, which are failing in GHA. Removing this test solves it, but I wanted to check in first in case I'm missing something: https://github.com/trufflesuite/truffle/blob/develop/packages/truffle/test/scenarios/library/api.js#L26
Ok to remove?

Reply:
No, I think this is being exposed so that users can pull it out and use it separately.

Reply:
So I wound up taking out the test, after discussing with gnidan. The repl manager doesn't exist anymore so that export isn't really meaningful anymore. Let me know if you want to discuss this more!